src/olympia/reviewers/utils.py

```diff
@@ -755,17 +765,17 @@ def process(self):
 
 class ReviewBase:
     def __init__(
-        self, request, addon, version, review_type, content_review=False, user=None
+        self,
+        *,
```

Comment:
What is the `*` used for in the class?

Reply:
It's to force the arguments that follow to be keyword-only arguments. Prevents you from accidentally using positional arguments, making calls more explicit.